tests/installation/install_and_reboot.pm

```diff
@@ -103,13 +103,22 @@ sub run() {
         last;
     }
 
-    # Upload logs before reboot
+    # Stop reboot countdown for e.g. uploading logs
     if (!get_var("REMOTE_CONTROLLER")) {
         do {
             send_key 'alt-s';
         } until (wait_still_screen(2, 4));
+
```

Comment:
please no changing back and forth of unrelated whitespace. Add some rules to perltidy if you prefer this and convince everybody